packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -932,6 +977,18 @@ export class BaseResolversVisitor<
       const realType = baseType.name.value;
       const parentType = this.schema.getType(parentName);
 
+      let contextName = this._fieldContextTypeMap[`${parentName}.${node.name}`]
+        ? this._fieldContextTypeMap[`${parentName}.${node.name}`].type
+        : 'ContextType';
+
+      for (const directive of node.directives) {
+        const name = directive.name as unknown as string;
```

Comment:
I need some help here, seemps that the type for `node.directives` is not correct!
Can I fix it?